stdlib/cmake/modules/StdlibOptions.cmake

```diff
@@ -99,6 +99,13 @@ option(SWIFT_STDLIB_EMIT_API_DESCRIPTORS
         "Emit api descriptors for the standard library"
         FALSE)
 
+option(SWIFT_STDLIB_BUILD_ONLY_CORE_MODULES
+       "Build only the core subset of the standard library,
+       ignoring additional libraries such as Concurrency, Distributed and StringProcessing.
+       This is an option meant for internal configurations that need to build
+       the standard libraries in chunks when constructing an SDK"
```

Comment:
I personally would suggest dropping that extra context, but that is just me.  At the very least, "internal" should be more explicit as "Apple".  This is a vendor specific option, which is something we _want_ to support, but, "internal" is confusing for a project which has more than one vendor.

Reply:
Unfortunately at the moment I would not have much time to consider more fundamental (re)work, but I will keep this observation in the back of my head.

Amended comments in 73d52987eb1c895f96c7bcd258748e37f5b60d5c .

Reply:
@edymtt - I'm not sure what fundamental work you had in mind, the comment is exactly the change :).  I was simply saying that having options for vendors to build as they need is something that the project should allow (aka, this change should go in). However, "internal configuration" is a bit confusing as there may be different vendors, so we should either say which vendor or simply not state that this is for internal use :).

Reply:
Ah, I misunderstood "dropping that extra context" as "separate those libraries out of the stdlib", not "dropping those lines in the description of the option". :(

I will still keep this in mind for next time -- I admit I tend to be a bit verbose.